pkg/base1/cockpit.js

```diff
@@ -1598,6 +1606,11 @@ function full_scope(cockpit, $, po) {
                 $.extend(cache.meta, msg.meta);
             } else if (msg.owner !== undefined) {
                 $(self).triggerHandler("owner", [ msg.owner ]);
+
+                if (track && owner)
+                    self.close();
```

Comment:
We guarantee that we'll never get an "owner" message with the same owner as previously sent right? If so we should just have a comment about that here.